dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/DynamicVirtualHostManager.java

```diff
@@ -216,19 +231,50 @@ public DynamicVirtualHost getVirtualHost(String name, WebContainer parent) {
         return vhost;
     }
 
+    @Reference(service = WsLocationAdmin.class,
+                    policy = ReferencePolicy.DYNAMIC,
+                    cardinality = ReferenceCardinality.OPTIONAL,
+                    policyOption = ReferencePolicyOption.GREEDY)
```

Comment:
Since you're now defining the WsLocationAdmin service and ScheduledExecutorService via annotations shouldn't we also clean up the bnd.bnd where these are defined?

Reply:
Does the policy/cardinality/policyOptions match the previous definition of these services without using the annotations?